### ISSUE TITLE
Did you specified the right host or port?

### ISSUE THREAD

**Issue by Ehsan125** [mylinuxhub/Kubernetes-#1]
I am directed here after seeing your tutorial on youtube, i have seen the solution on other forums as well and applied those as well, but i am not getting rid off from this error, my working environment details are as follows:

Host Operating System : Windows 11
VM Virtual Box guest Operating System : Ubuntu 22.04 LTS

`sudo kubeadm init `
the above command succefully generated the token and also the `admin.config`  but while running the the following command after placing the config file on correct folder as you described still does not help in removing the error: 

`sudo kubectl get nodes`

The master node is in UBUNTU 22.04,  i have turn off the swap space using the following commands: 

```
sudo swapoff -a
sudo sed -i '/ swap / s/^/#/' /etc/fstab

```
I have followed the following steps in a sequence, i am attaching the screenshots, i am hoping that you will anticipate:
![1-status kubectl](https://user-images.githubusercontent.com/45161966/167295440-6a4855d1-4c44-4fcc-8910-b8084491012f.png)
![2-key succesfully generated](https://user-images.githubusercontent.com/45161966/167295456-1f1fffaf-3397-457a-b8c4-8ca6be170b18.png)
![4](https://user-images.githubusercontent.com/45161966/167295609-c8cbeee7-ead6-4a8f-af49-9c601d44f01f.png)
![3-key succesfully generated](https://user-images.githubusercontent.com/45161966/167295460-131453e2-e525-4134-9799-2aae3f555d2a.png)
 

pleease needful help is required if you can my error still persist which is as follows: 

`The connection to the server localhost:8080 was refused - did you specify the right host or port?`

i dont know where i am doing wroing i have tried almost every possible combination of solution but all in vain.

**Comment by mylinuxhub** [comment]
Hi,

Can you follow the below link for steps in the Kubernetes lab?

https://github.com/mylinuxhub/Kubernetes-/blob/main/How%20to%20Setup%20Kubernetes%20Cluster%20on%20Google%20Cloud


Can you cat /etc/default/kubelet file and paste the output here.

**Comment by Ehsan125** [comment]
where i am doing wrong, can you please specify ? i have followed all the steps , i am still not getting rid off

**Comment by Ehsan125** [comment]
![image](https://user-images.githubusercontent.com/45161966/167364928-3ad7635f-347b-4f39-bfb6-fab65d807190.png)
 
i have just only these files in `/etc/default` directory , i am not seeing the `kubelet file` in this directory

**Comment by mylinuxhub** [comment]
Create the below file.

vi /etc/docker/daemon.json

#add below entry. save and exit, after that try to restart.
{
      "exec-opts": ["native.cgroupdriver=cgroupfs"],
      "log-driver": "json-file",
      "log-opts": {
      "max-size": "100m"
   },
       "storage-driver": "overlay2"
       }

**Comment by Ehsan125** [comment]
I have already made this file and ran the following commands as you directed in the tutorial: 

```
systemctl daemon-reload
systemctl restart kubelet
systemctl status  kubelet
```

but still the error persist. Would you suggest me that if i run the 99 steps in the same sequence as you mentioned in the below file 

`How to Setup Kubernetes Cluster on Google Cloud`

should i do all these 99 steps and then i will get rid off, i think doing so does not make any sense to me. please provide your anticipation please

**Comment by mylinuxhub** [comment]
sometimes steps also create problems. I can suggest you go for a fresh installation and follow the steps. If you want I will help you with a step lab.

**Comment by Ehsan125** [comment]
Fresh installation of what? Docker? Or kubeadm? Or Whole VM? Please guide i will do that, Last whole week i had spent on this stuff

**Comment by mylinuxhub** [comment]
Build a Fresh master vm and do a fresh installation. Open the bridge call, I will join and help you.

**Comment by Ehsan125** [comment]
May i share my skype id ? if we can arrange a call on your convenience if possible please, it is great headache for me from last 1 week, please

**Comment by mylinuxhub** [comment]
sure, Share with me I will try to connect this Friday.

**Comment by Ehsan125** [comment]
Thank you for your response, my skype id is 
ehsanirshad125@outlook.com
Can we set up meeting after 14 hours? Let me reach office please? My whatsapp number is +923315586514, please reach me , thank you

**Comment by Ehsan125** [comment]
are you available please?

**Comment by mylinuxhub** [comment]
Today I am not available. Tomorrow morning we can connect at 9:00 AM

**Comment by Ehsan125** [comment]
Yes dear, sure, Thank you for providing me assistence out of the way.

**Comment by mylinuxhub** [comment]
I am available now. if you want Open the bridge call

**Comment by Ehsan125** [comment]
Dear i am sorry, i was waiting for your ping message on Skype or whatsapp. Please ping me there, so that we can arrange then. My whole setup is in my office. Can you ping me right now, and we will setup actual meeting accordingly later?

**Comment by LokendraDevOps** [comment]
Yes
